pkg/model/model.go

```diff
@@ -15,6 +15,7 @@ type BaseModel struct {
 
 type Job struct {
 	BaseModel
+	Provider    string `json:"provider"`
```

Comment:
Maybe use int instead so that there is less chance of a mistake and also more performant?

In the scraper package, you can declare a new type of Provider and its int value, then declare this as the custom type Provider instead.

```go
package scraper

type Provider int

const (
    JobCenter Provider = 1
    Bruneida Provider = 2
)
```

Reply:
giving it type doesn't work

```go
type Provider int

const (
    JobCenter Provider = 1
)

job := models.Job{Providder: JobCenter} // <-- error

```